HIP/hip-1.md

```diff
@@ -74,11 +75,15 @@ Once the champion has discovered with the Hedera community the acceptability of
 
 The standard HIP workflow is:
 
-* You, the HIP author, fork the HIP repository, and create a file named hip-0000-my-feature.md (where “my-feature” is descriptive) that contains your new HIP. Use 0000 as your draft HIP number. This file should be created from the HIP template.
+* You, the HIP author, fork the HIP repository, and create a file named `hip-0000-my-feature.md` (where "my-feature" is descriptive) that contains your new HIP. Use 0000 as your draft HIP number. This file should be created from the HIP template.
+
+* In the "Type" header field, enter "Standards Track", "Informational", or "Process" as appropriate, and for "Status" field enter "Draft".
```

Comment:
Did we want the status to be "WIP" first and when merged HIP editors update to  "Draft"

Reply:
Good call, updating.

Reply:
Actually, deprecated "WIP" status in favour of "Draft" to simplify the process